diff --git a/build/index.js b/build/index.js
index 6f0ef67ec5d86c98e02f03293ce98ca15056c5df..47697746f5215892ece95fcfc35983fcaf1c0eb3 100644
--- a/build/index.js
+++ b/build/index.js
@@ -4,13 +4,14 @@ import Keyv from "keyv";
 import pTimeout from "p-timeout";
 import QuickLRU from "quick-lru";
 import { v4 as uuidv4 } from "uuid";
+import { fetch } from 'cross-fetch';
+globalThis.fetch = fetch;
 
 // src/types.ts
 var ChatGPTError = class extends Error {
 };
 
 // src/fetch.ts
-var fetch = globalThis.fetch;
 if (typeof fetch !== "function") {
   throw new Error("Invalid environment: global fetch not defined");
 }